packages/jobs/lib/temporal.ts

```diff
@@ -34,9 +34,14 @@ export class Temporal {
             key = await fs.readFile(`/etc/secrets/${this.namespace}.key`);
         }
 
+        const temporalAddress = process.env['TEMPORAL_ADDRESS'];
+        if (!temporalAddress) {
+            throw new Error('TEMPORAL_ADDRESS missing from env var');
+        }
+
         try {
             const connection = await NativeConnection.connect({
-                address: process.env['TEMPORAL_ADDRESS'] || 'localhost:7233',
+                address: temporalAddress,
```

Comment:
does that mean I now need the env var for my local setup? :)

Reply:
I am ok with that. We have some defaults that are aimed at making our local dev life easier but they make detecting bad config in prod more silent.